framework/db/ActiveRecord.php

```diff
@@ -110,12 +110,15 @@ class ActiveRecord extends BaseActiveRecord
      *
      * @param boolean $skipIfSet whether existing value should be preserved.
      * This will only set defaults for attributes that are `null`.
+     * @param boolean $skipExpressions whether to ignore \yii\db\Expression values.
      * @return $this the model instance itself.
      */
-    public function loadDefaultValues($skipIfSet = true)
+    public function loadDefaultValues($skipIfSet = true, $skipExpressions = true)
```

Comment:
I don't think so. At least it's just a new argument and that's why the default is true.